test/sslapitest.c

```diff
@@ -1568,7 +1570,9 @@ int test_main(int argc, char *argv[])
 #ifndef OPENSSL_NO_TLS1_3
     ADD_TEST(test_keylog_no_master_key);
 #endif
+#ifndef OPENSSL_NO_TLS1_2
     ADD_TEST(test_early_cb);
+#endif
```

Comment:
No - this test *only* works if we have v1.2, i.e. if 1.2 is disabled we shouldn't run it. It uses some 1.2 only ciphersuites.